ewlegal-opinion.sty

```diff
@@ -1,23 +1,43 @@
-\ProvidesPackage{engwales-opinion}[2021-02-13 Package for pleading creation in England/Wales styles]
+%% TODO Stock beginning text, copyrights and project info and all that
 
-\RequirePackage[explicit]{titlesec}
-\RequirePackage{enumitem}
-\RequirePackage{fancyhdr}
+\ProvidesPackage{ewlegal-opinion}[2021-02-13 England and Wales Opinion Legal Template]
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typeset Elements
+%%%%%
```

Comment:
are we ready to embrace this as a standard in this project? i think the spacing and the _two_ separators are a bit too much. but as long as everyone agrees.

Reply:
Yea, it's up to everyone else. I'm aware it's a bit much for most but I really like readability.

Reply:
ah no i'm not saying get rid of all of this. i too like separators. i just think this is a bit too much. (e.g., one empty line + one separator might be enough.)